OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -91,11 +95,13 @@ bool AttackTargets(Actor self, IEnumerable<Actor> targets)
 				}
 			});
 
+			var barrel = armament.CheckFire(self, facing, target);
```

Comment:
Removing the warhead makes the worm circle around targeted actors endlessly without ever swallowing them.